src/nxdoc/nuxeo-server/installation/kubernetes-helm-chart.md

```diff
@@ -249,146 +249,146 @@ The parameters of the `minio`, `mongodb`, `opensearch` and `kafka` charts can be
-| `postgresql.enabled` | Enable PostgreSQL backend for Nuxeo | `false` |
-| `postgresql.host` | Host for PostgreSQL connection | `""` |
-| `postgresql.port` | Post for PostgreSQL connection | `5432` |
-| `postgresql.auth.username` | PostgreSQL authentication username | `""` |
-| `postgresql.auth.password` | PostgreSQL authentication password | `""` |
-| `postgresql.auth.existingSecret` | Existing secret with PostgreSQL credentials (keys: `postgresql-username`, `postgresql-password`), overrides `username` and `password` | `""` |
-| `postgresql.initContainer.repository` | Image name for PostgreSQL connection init container | `busybox` |
-| `postgresql.initContainer.tag` | Image tag for PostgreSQL connection init container | `latest` |
-| `postgresql.initContainer.imagePullPolicy` | Image pull policy for PostgreSQL connection init container | `IfNotPresent` |
-| `elasticsearch.enabled` | Enable Elasticsearch for Nuxeo | `false` |
-| `elasticsearch.protocol` | Protocol for Elasticsearch connection | `http` |
-| `elasticsearch.host` | Host for Elasticsearch connection | `""` |
-| `elasticsearch.port` | Post for Elasticsearch connection | `9200` |
-| `elasticsearch.clusterName` | Elasticsearch cluster name | `elasticsearch` |
-| `elasticsearch.indexNumberOfReplicas` | Elasticsearch number of replicas | `0` |
-| `elasticsearch.restClient.socketTimeoutMs` | Elasticsearch REST client socket timeout in ms | `300000` |
-| `elasticsearch.restClient.connectionTimeoutMs` | Elasticsearch REST client connection timeout in ms | `300000` |
-| `elasticsearch.auth.enabled` | Enable Elasticsearch authentication | `false` |
-| `elasticsearch.auth.username` | Elasticsearch authentication username | `""` |
-| `elasticsearch.auth.password` | Elasticsearch authentication password | `""` |
+| Parameter | Description                                                                                                                                    | Default |
+| --------- |------------------------------------------------------------------------------------------------------------------------------------------------| --------|
+| `image.repository`| Nuxeo image name                                                                                                                               | `docker-private.packages.nuxeo.com/nuxeo/nuxeo` |
+| `image.tag` | Nuxeo image tag                                                                                                                                | `2023` |
+| `image.pullSecrets` | Nuxeo image registry secret names                                                                                                              | `[]` |
+| `image.pullPolicy` | Nuxeo image pull policy                                                                                                                        | `IfNotPresent` |
```

Comment:
Are we sure we want such a formatting change? It's a lot of spaces in the editor :)

Reply:
looks like it's the default md formatting in Intellij